VSRAD.Package/DebugVisualizer/GroupIndexSelector.cs

```diff
@@ -151,6 +151,8 @@ public void Update()
             ErrorsChanged?.Invoke(this, new DataErrorsChangedEventArgs(nameof(Z)));
         }
 
+        public void SetGroupIndex(int index) => X = (uint)index;
```

Comment:
I think this method can be safely removed and all usages replaced with `GoToGroup`, which (in theory, I haven't done much testing) should also work when 3D NDRange is enabled.